frontend/packages/console-shared/src/constants/common.ts

```diff
@@ -36,6 +36,7 @@ export const API_DISCOVERY_RESOURCES_LOCAL_STORAGE_KEY = `${STORAGE_PREFIX}/api-
 export const COMMUNITY_PROVIDERS_WARNING_LOCAL_STORAGE_KEY = `${STORAGE_PREFIX}/community-providers-warning`;
 export const DEV_CATALOG_FILTER_KEY = `${STORAGE_PREFIX}/dev-catalog-filters`;
 export const PINNED_RESOURCES_LOCAL_STORAGE_KEY = `${STORAGE_PREFIX}/pinned-resources`;
+export const COLUMN_MANAGEMENT_LOCAL_STORAGE_KEY = `${STORAGE_PREFIX}/column-management`;
```

Comment:
Maybe call this "table columns" instead of "column management"? I think that makes it more clear what it's for.

```suggestion
export const TABLE_COLUMNS_LOCAL_STORAGE_KEY = `${STORAGE_PREFIX}/table-columns`;
```

Reply:
done